models/demos/mamba/tests/test_mamba_ssm.py

```diff
@@ -51,6 +56,8 @@ def test_mamba_ssm_inference(
 
     reference_model = MambaDecode.from_pretrained(model_version)
     reference_model.args.batch_size = batch
+    reference_model.args.seq_len = 1
+    reference_model.args.mode = ModelMode.DECODE
```

Comment:
Adding a 'mode' to the reference model is kinda confusing since it doesn't support both. Can we make the model mode a member of MambaTT instead? We could pass it during initialization and default it to decode.

Reply:
Maybe combine prefill_model.py and decode_model.py then?